topology_builder/pom.xml

```diff
@@ -33,18 +33,18 @@
     <dependency>
       <groupId>org.apache.storm</groupId>
       <artifactId>storm-core</artifactId>
-      <version>0.9.4</version>
```

Comment:
I'm cool to revert the changes in pom.xml, it is just that my company has this combination in our production and we are not gonna upgrade Storm very soon... So I figured it is best to alter the settings in my environment. Storm 0.9.4 is totally fine and from my experience Kafka 0.8.2.1 has better management ability than 0.8.1. There has not been any broken changes seen in 0.8.2.1 and we are currently using both 0.8.1 and 0.8.2 in production environment.